crates/shielded_token/src/storage_key.rs

```diff
@@ -172,3 +186,51 @@ pub fn masp_total_rewards() -> storage::Key {
         .push(&MASP_TOTAL_REWARDS.to_owned())
         .expect("Cannot obtain a storage key")
 }
+
+/// The owner of a shielded action transfer, could be a proper address or the
+/// minted subkey
+pub enum ShieldedActionOwner<'key> {
```

Comment:
I believe this type is not part of the codebase anymore

Reply:
ah nice, rm'd in https://github.com/anoma/namada/compare/00288176411fe50b3481841ea7cf3c41b0c59370..7f87875237a7c14549f2e1e38bd06517bca7196d